wolfcrypt/src/port/intel/quickassist.c

```diff
@@ -557,6 +557,12 @@ int IntelQaOpen(WC_ASYNC_DEV* dev, int devId)
         return ASYNC_INIT_E;
     }
 
+    if (devId >= g_numInstances) {
+        printf("devId %d out of range. Number of instances=%u.\n", devId,
```

Comment:
Please use: `fprintf(stderr, "IntelQA: devId %d exceeds number of instances %u\n", devId, g_numInstances);`

Reply:
Done